test/blocks/commerce/commerce.test.js

```diff
@@ -0,0 +1,96 @@
+import { expect } from '@esm-bundle/chai';
+import { readFile } from '@web/test-runner-commands';
+import {
+  formatOfferDetails,
```

Comment:
This function doesn't exist (anymore?) and breaks the test. Please remove it.

Reply:
I think the initial function was renamed to `filterOfferDetails`, please adjust.

Reply:
Done ✅